doc/uftrace-graph.md

```diff
@@ -28,6 +28,10 @@ GRAPH OPTIONS
     Multiple fields can be set by using comma.  Special field of 'none' can be
     used (solely) to hide all fields.  Default is 'total'.  See *FIELDS*.
 
+--task
+:   Print task graph instead of normal function graph.  The each node in the
+    output shows process(with an asterisk) or thread(printed in green color).
```

Comment:
It'd be nice if you add an example and descriptions for fields.

Reply:
I added more examples.